ci/test_windows.bat

```diff
@@ -8,7 +8,16 @@ call venv\scripts\activate
 pip install -U snowflake-connector-python
 
 cd %GITHUB_WORKSPACE%
-gpg --quiet --batch --yes --decrypt --passphrase=%PARAMETERS_SECRET% --output parameters.json .github/workflows/parameters_aws.json.gpg
+
+if "%CLOUD_PROVIDER%"=="AZURE" (
+  set ENCODED_PARAMETERS_FILE=.github/workflows/parameters_azure.json.gpg
+) else if "%CLOUD_PROVIDER%"=="GCP" (
+  set ENCODED_PARAMETERS_FILE=.github/workflows/parameters_gcp.json.gpg
+) else (
```

Comment:
Nit: Right now we only support 3 providers, so last one implicitly become AWS. But way to check for AWS and use the aws json file and in final else fail the build with "unknown provider".

Reply:
I added explicit check for AWS for Windows and also the same logic for Linux and Mac